argutils/viz.py

```diff
@@ -174,3 +205,23 @@ def nx_get_dot_pos(G, reverse_x_axis=None):
         if n.get_pos() and n.get_name().isdigit()
     }
 
+
+def get_edge_spans(G):
+    """
+    Returns a list of length "num_edges" in G, the graphical representation of the
+    tree sequence, where each entry is the genomic span of the corresponding edge in the
+    tree sequence divided by the total sequence length.
+    """
+    edge_spans = list()
+    for edge in G.edges():
+        edge_data = G.get_edge_data(edge[0], edge[1])
+        edge_spans.append(edge_data["intervals"][0][1] - edge_data["intervals"][0][0])
```

Comment:
This is assuming you've only one interval per edge though. Should be something like
```
total_span = sum(right - left for (left, right) in edge_data["intervals"])
```